bin/wasm-node/javascript/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Light client that connects to Polkadot and Substrate-based blockchains",
   "author": "Parity Technologies <admin@parity.io>",
   "license": "GPL-3.0-or-later WITH Classpath-exception-2.0",
+  "type": "module",
```

Comment:
Why put this back? If it's to fix the test, can you not rename the tests to `.mjs`?

Reply:
Yeah I can.  I mean - its the same thing. will do the renaming